java/benchmarks/src/main/java/glide/benchmarks/clients/glide/GlideAsyncClient.java

```diff
@@ -0,0 +1,61 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.benchmarks.clients.glide;
+
+import glide.api.RedisClient;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.RedisClientConfiguration;
+import glide.benchmarks.clients.AsyncClient;
+import glide.benchmarks.utils.ConnectionSettings;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+/** A Glide client with async capabilities */
+public class GlideAsyncClient implements AsyncClient<String> {
+    private RedisClient redisClient;
+
+    @Override
+    public void connectToRedis(ConnectionSettings connectionSettings) {
+        if (connectionSettings.clusterMode) {
+            throw new RuntimeException("Use client GlideAsyncClusterClient");
```

Comment:
But there is no `GlideAsyncClusterClient`.
Implement it here, like it is done for lettuce

Reply:
I will push this in a second...